### DISCUSSION TITLE
Release process - confusion

### DISCUSSION THREAD

**Discussion by naturzukunft** [eclipse-rdf4j/rdf4j#4082] [General]
Hi there,
because https://github.com/eclipse/rdf4j/pull/4009 is merged into eclipse/main, I was expecting, that it is already in https://mvnrepository.com/artifact/org.eclipse.rdf4j/rdf4j-spring-boot-sparql-web/4.0.4 but it isn't. How is the process to get things available in a release ?
Fredy

**Comment by naturzukunft** [comment]
Ah, i see the 4.1.0 MS Release, has a comment "Improvements to the Spring Components".

**Comment by hmottestad** [comment]
We tag the issues with the version that they are targeting. https://github.com/eclipse/rdf4j/issues/4004
